app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -19,7 +19,7 @@ def textual_group_relationships
       _("Relationships"),
       %i[
         ems_infra network_manager availability_zones host_aggregates cloud_tenants flavors
-        security_groups instances images cloud_volumes orchestration_stacks storage_managers cloud_databases
+        security_groups placement_groups instances images cloud_volumes orchestration_stacks storage_managers cloud_databases
```

Comment:
```suggestion
        security_groups placement_group instances images cloud_volumes orchestration_stacks storage_managers cloud_databases
```

I believe this name needs to match the `textual_placement_group` method below. (I got confused and mistakenly left a comment earlier suggesting to incorrectly change `placement_group`->`placement_groups` in the vm textual summary module, deleted now).

Reply:
... actually I think it should be `placement_groups` here 😄! But  the method should also be named `textual_placement_groups`. I was able to get it working properly with the following patch:
https://github.com/jaywcarman/manageiq-ui-classic/commit/74a7fc15049b20fa204a90b9a2cb90b3a3200398